athena-snowflake/src/test/java/com/amazonaws/athena/connectors/snowflake/SnowflakeRecordHandlerTest.java

```diff
@@ -73,7 +73,7 @@ public void setup()
         this.connection = Mockito.mock(Connection.class);
         this.jdbcConnectionFactory = Mockito.mock(JdbcConnectionFactory.class);
         Mockito.when(this.jdbcConnectionFactory.getConnection(nullable(JdbcCredentialProvider.class))).thenReturn(this.connection);
-        jdbcSplitQueryBuilder = new SnowflakeQueryStringBuilder("`");
+        jdbcSplitQueryBuilder = new SnowflakeQueryStringBuilder("`", new SnowflakeFederationExpressionParser("`"));
```

Comment:
this is not the same quote char used in athena-snowflake/src/test/java/com/amazonaws/athena/connectors/snowflake/SnowflakeQueryStringBuilderTest.java

Reply:
resolved.